package.json

```diff
@@ -9,6 +9,7 @@
     "lint:fix": "eslint --fix *.js lib tests",
     "prepare": "husky install",
     "test": "npm run unit && npm run integration && npm run versioned",
+    "third-party-updates": "oss third-party manifest --includeOptDeps && oss third-party notices --includeOptDeps && git add THIRD_PARTY_NOTICES.md third_party_manifest.json",
```

Comment:
this repo doesn't have optional deps so you could remove that flag or not worry about it. if you do remove i think you'll have to manually remove the `includeOptDeps` in the manifest before re-running

Reply:
i'd thought about that but then left in case we ever added an optional dep so we dont need to remember. but thinking on again, the chance of that is extremely low. we shouldn't really be adding optional deps to external instrumentation like this. so i think i will rip it out.

Reply:
done